RecoTracker/TkTrackingRegions/src/OuterHitPhiPrediction.h

```diff
@@ -20,7 +20,7 @@ class dso_internal OuterHitPhiPrediction {
       float originRBound) 
     : thePhiAtVertex(phiAtVertex), theCurvature(curvature),
       theOriginRBound (originRBound) {
-       assert(theCurvature.max()>0);
+	//       assert(theCurvature.max()>0);
```

Comment:
@VinInn @slava77 
See also the comments made very recently at https://github.com/cms-sw/cmssw/pull/13819
which added these "assert"s.